### PR TITLE
feat: Implement ReleaseCreated

### LINKED ISSUE #61: Handle release events
Docs https://developer.github.com/v3/activity/events/types/#releaseevent

# Implementation plan

- [x] Allow `release` events on webhook `ping`
- [x] Add `ReleaseCreated` to the `reactionBuild` function
- [x] Handle `release` events, only when they're published
- [x] Unit-test the reaction
- [x] Test the reaction with integration tests

### PR DESCRIPTION
**Pull request by orestes**
Resolves #61﻿

### CODE REVIEW

File: test/reactions/github/release-created.spec.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+			const payload = new ReleaseCreatedPayloadBuilder()
+				.with({ action: 'published' })
+				.getInstance();
+
+			expect(subject.canHandle({ payload, event })).toEqual(true);
+		});
+	});
+
+	describe('#getTwitchChatMessage', () => {
```
Comment (SantiMA10) [inline test/reactions/github/release-created.spec.ts:45]: I think you miss an `it` here 🤔

File: test/reactions/github/release-created.spec.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+		const subject = new ReleaseCreated(twitchChat, streamLabs);
+		const payload = new ReleaseCreatedPayloadBuilder().with({ action: 'published' }).getInstance();
+
+		expect(subject.getTwitchChatMessage({ payload })).toEqual(
+			`streamdevs/webhook version 1.0.0 has just been released 🚀! Check it out http://github.com/streamdevs/webhook/releases/1.0.0`,
+		);
+	});
+
+	describe('#getStreamLabsMessage', () => {
```
Comment (SantiMA10) [inline test/reactions/github/release-created.spec.ts:54]: Another `it` missing here 🤔

File: test/routes/github/release-created.spec.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import { initServer } from '../../../src/server';
+import { getConfig } from '../../../src/config';
+import { StreamLabs } from '../../../src/services/StreamLabs';
+import { TwitchChat } from '../../../src/services/TwitchChat';
+import { ReleaseCreatedPayloadBuilder } from '../../builders/release-created-payload-builder';
+import { ReleaseCreatedPayload } from '../../../src/schemas/github/release-created-payload';
+import { WebhookResponse } from '../../../src/schemas/webhook-response';
+
+describe('/github', () => {
```
Comment (SantiMA10) [inline test/routes/github/release-created.spec.ts:9]: Can we add the HTTP method here?

Comment (orestes) [reply test/routes/github/release-created.spec.ts]: Fixed in https://github.com/streamdevs/webhook/pull/148/commits/c611c517113adfd021ff51e8858c8a5382d0decf even thought GitHub is not updating this diff

**Pull request merged**